### PR TITLE
printfs to zowelogs

### PR DESCRIPTION
**Pull request by sakeerthy**
WIP
Signed-off-by: sakeerthy <sakeerthy@gmail.com>

### CODE REVIEW

File: c/omvsService.c [hunk 1]
```diff
@@ -151,7 +151,7 @@ static int parseOMVSSegment(RadminAPIStatus status, const RadminCommandOutput *r
   if (status.racfRC != 0)
   {
 #ifdef DEBUG
-    printf("Error: reason: %d, return: %d\n", status.racfRSN, status.racfRC);
+    zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_SEVERE, "Error: reason: %d, return: %d\n", status.racfRSN, status.racfRC);
```
Comment (1000TurquoisePogs) [inline c/omvsService.c]: Dont double up ifdef with zowelog. zowelog is good here.

File: c/serverStatusService.c [hunk 2]
```diff
@@ -169,7 +169,7 @@ int respondWithServerEnvironment(HttpResponse *response, ServerAgentContext *con
   safeFree(buffer, bufferlen);
   if (rc > 0) {
     respondWithError(response, HTTP_STATUS_BAD_REQUEST, "Unable to fetch from RMF data interface service");
-    printf("Failed to fetch RMF data, RC = %d\n", rc);
+    zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_INFO, "Failed to fetch RMF data, RC = %d\n", rc);
```
Comment (1000TurquoisePogs) [inline c/serverStatusService.c]: warning, not info

File: c/zosDiscovery.c [hunk 3]
```diff
@@ -198,12 +199,12 @@ static void serveCICSData(ZOSModel         *model,
   }
   jsonStartArray(p,"rows");
   SoftwareInstance *subsystemList = model->cicsSubsystems;
-  printf("before CICS list subsystemList 0x%x\n",subsystemList);
+  zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_INFO, "before CICS list subsystemList 0x%x\n",subsystemList);
```
Comment (1000TurquoisePogs) [inline c/zosDiscovery.c]: debug, not info.

File: c/zosDiscovery.c [hunk 4]
```diff
@@ -198,12 +199,12 @@ static void serveCICSData(ZOSModel         *model,
...
   while (subsystemList){
 
     SoftwareInstance *info = subsystemList;
     char *name = info->bestName;
-    printf("CICS list elt=0x%x, best=%s type=0x%llx\n",info,info->bestName,info->type);
+    zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_INFO, "CICS list elt=0x%x, best=%s type=0x%llx\n",info,info->bestName,info->type);
```
Comment (1000TurquoisePogs) [inline c/zosDiscovery.c]: debug

File: c/zosDiscovery.c [hunk 5]
```diff
@@ -299,7 +300,7 @@ static int serveSubsystemData(HttpService *service,
   zowelog(NULL, LOG_COMP_DISCOVERY, ZOWE_LOG_DEBUG2, "should serve discovery stuff\n");
   ZOSModel *model = discoveryServiceState->model;
   SoftwareInstance *instanceList = zModelGetSoftware(model);
-  printf("done with discovery, or reusing cache \n");
+  zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_INFO, "done with discovery, or reusing cache \n");
```
Comment (1000TurquoisePogs) [inline c/zosDiscovery.c]: debug

File: c/zosDiscovery.c [hunk 6]
```diff
@@ -323,7 +324,7 @@ static int serveSubsystemData(HttpService *service,
   }
   jsonEnd(p);
   finishResponse(response);
-  printf("done with ss response\n");
+  zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_INFO, "done with ss response\n");
```
Comment (1000TurquoisePogs) [inline c/zosDiscovery.c]: debug

File: c/zosDiscovery.c [hunk 7]
```diff
@@ -382,7 +383,7 @@ static int serveTN3270Data(HttpService *service,
     findSessions(context,SESSION_KEY_TYPE_IP6_STRING,0,luname);
   } else{
     findSessions(context,SESSION_KEY_TYPE_ALL,0,luname);
-    printf("get'em all\n");
+    zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_INFO, "get'em all\n");
```
Comment (1000TurquoisePogs) [inline c/zosDiscovery.c]: debug

File: c/zosDiscovery.c [hunk 8]
```diff
@@ -418,7 +419,7 @@ static int serveTN3270Data(HttpService *service,
   jsonEnd(p);
   freeDiscoveryContext(context);
   finishResponse(response);
-  printf("done with system response\n"); 
+  zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_INFO, "done with system response\n");
```
Comment (1000TurquoisePogs) [inline c/zosDiscovery.c]: debug

File: c/zosDiscovery.c [hunk 9]
```diff
@@ -486,30 +487,30 @@ static int serveDiscoveryData(HttpService *service, HttpResponse *response){
...
   } else{
-    printf("unknown zosDiscovery first level name=%s\n",firstLevelName);
+    zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_WARNING, "unknown zosDiscovery first level name=%s\n",firstLevelName);
   }
   return 0;
 }
 
 int zosDiscoveryServiceInstaller(DataService *dataService, HttpServer *server){
-  printf("%s begin\n", __FUNCTION__);
+  zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_INFO, "%s begin\n", __FUNCTION__);
```
Comment (1000TurquoisePogs) [inline c/zosDiscovery.c]: debug

File: c/zosDiscovery.c [hunk 10]
```diff
@@ -523,7 +524,7 @@ int zosDiscoveryServiceInstaller(DataService *dataService, HttpServer *server){
     CrossMemoryServerName *priviligedServerName = getConfiguredProperty(server, HTTP_SERVER_PRIVILEGED_SERVER_PROPERTY);
     state->model = makeZOSModel(priviligedServerName);
     dataService->extension = state;
-    printf("%s end\n", __FUNCTION__);
+    zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_INFO, "%s end\n", __FUNCTION__);
```
Comment (1000TurquoisePogs) [inline c/zosDiscovery.c]: debug

File: c/zss.c [hunk 11]
```diff
@@ -1014,15 +1014,15 @@ int initializeJwtKeystoreIfConfigured(JsonObject *const serverConfig,
 
 int main(int argc, char **argv){
   if (argc == 1) { 
-    printf("Usage: zssServer <path to server.json file>\n");
+    zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_INFO, "Usage: zssServer <path to server.json file>\n");
```
Comment (1000TurquoisePogs) [inline c/zss.c]: warning

File: c/zosDiscovery.c (hunk 7 above)
Comment (DivergentEuropeans) [reply c/zosDiscovery.c]: image

**Pull request merged**